src/models/task.ts

```diff
@@ -0,0 +1,7 @@
+export interface ITask {
+  status?: boolean;
+  name?: string;
+  id?: string;
+  categoryID?: string;
+  order?:number;
+}
```

Comment:
All files directly related to app development should be under the `/src/app` directory. The `models` folder should not be out here.

Also, I don't recommend the file structure you're adopting right now.

Here's an example of the file structure I used in one of my projects.
```
/ src
-/ app
--/ core 
---/ services <-- your generated services
----/ list.service.ts
---/ interfaces <-- your interface types for Task and Categories, etc.
---/ constants <-- your enums, sample data and other constants
--/ (top-level modules or components) <-- like dashboard etc.
--/ shared <-- if a component is used by more than one component/module, it should be placed here
---/ (shared components) 
--/ app.component.ts
--/ app.module.ts
--/ app-routing.module.ts
```